### DISCUSSION TITLE
Faulty 6-channel board

### DISCUSSION THREAD

**Discussion by smeisner** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#72] [Q&A] [answered]
I also sent the following to John via email with no reply. I know there's a large backlog that's causing the delay, but my unit (that was part of the backlog) appears to be faulty.

> I have a CT on 1 line in my breaker box connected to CT1 port. I then flashed the NodeMCU with the following code via ESPHome in HomeAssistant:
https://github.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/blob/master/Software/ESPHome/6chan_energy_meter_main_board.yaml
It’s up and running, but I am getting nothing on CT1 – 3, Freq A and Volts A. I moved the CT over to CT6 jack and that is working perfectly (currently reading 2.37 Amps with a heater running and Freq B is 60Hz and Volts B is 120.4 V).
> 
> How should I troubleshoot this?

Additional details:
> -	I am using the recommended Jameco Reliapro adapter (https://www.amazon.com/gp/product/B00B886CAA)
> -	The only populated port in that screen shot is CT6, so ignore the values for CT4 & 5

After reviewing the schematic, it seems like the CS is not being asserted to enable chip #1. 

I am bringing this discussion to this forum for two reasons; It seems like there is more activity here than via email and has anyone else seen this? And if so, how did you troubleshoot & fix?

**Comment by CircuitSetup** [comment, marked as answer]
Sorry I didn't get back to you sooner. It sounds like there is something wrong with a connection to the first energy metering IC. 

I just installed a return/warranty system on https://circuitsetup.us under my account. If you can open a ticket there, I'll get you a replacement ASAP.

**Reply by smeisner** [reply]
I just followed every link I could find on your site and did not see anything related to returns or warranty support. Have you deployed this new web page yet?

**Reply by CircuitSetup** [reply]
Sorry, you should see an option for it under your orders: https://circuitsetup.us/my-account/order-list/

**Reply by smeisner** [reply]
nope...this is the first place I went to (as it was the most obvious). I see a link to the order, a "View" button and an "Invoice" button. That is it. And the order details do not have a returns element. I also tried a different browser and got the same thing.

**Reply by CircuitSetup** [reply]
humm, I must be missing something. How about this page: https://circuitsetup.us/my-account/request-warranty/
If you have your order number I can put it in manually as well.

**Reply by smeisner** [reply]
My order number is 215605.

If I click that last link, I do see a "Request Warranty/Return" page, but then it says "No order selected." There is no place to request the return.

**Reply by smeisner** [reply]
btw...in the future, if you want someone external to test out you site, feel free to ping me (you have my email). I do a lot of this sort of config work and having someone external to validate things is useful sometimes.

**Reply by CircuitSetup** [reply]
Sorry for the confusion, and thanks for the offer. I'm asking the vendor what I'm doing wrong with the plugin.

I put in the request manually so it's in the system. I'll get you a replacement in the mail today.

**Reply by smeisner** [reply]
Thanks for the support John!! It is this type of response that keep customers around!!

**Reply by smeisner** [reply]
So I see you created the USPS label over 2 days ago, but have yet to hand it off to USPS. Now I see on your site "We are moving! Orders will not be shipped between 1/11-1/18. Thank you for your patience!" Does this mean I am looking at a couple of weeks to replace the defective unit I have?

**Reply by CircuitSetup** [reply]
> So I see you created the USPS label over 2 days ago, but have yet to hand it off to USPS. 

It went out in yesterday's mail. I'm not sure why it's not showing up on the USPS site yet.

**Reply by smeisner** [reply]
OK, thank you for the confirmation. I have my breaker panel wide open awaiting this shipment, so I want to either close it up or wait for the new board.

**Reply by smeisner** [reply]
Just to close this out, I received the replacement board, plugged in the existing NodeMCU & CTs and everything worked perfectly!! I got to thinking about what the problem could be and (log story short), the first ATMEL chip was rotated 180deg. I followed the traces in the schematic and that lead to the realization. If you compare the pic from your Github project;

https://raw.githubusercontent.com/CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter/master/Images/6-channel_v1.3.jpg

with the pic of my (bad) board:
https://photos.app.goo.gl/xhcBcCatxGB5ffh59

it is clearly incorrect. I realize the first pic is of a v1.3 and mine is v1.4, but I also traced the circuit with a multi-meter to prove it is wrong.
